_std/packets.dm

```diff
@@ -7,22 +7,44 @@ proc/check_for_radio_jammers(atom/source)
 		if (IN_RANGE(source, A, RADIO_JAMMER_RANGE))
 			return TRUE
 
-#define MAKE_DEFAULT_RADIO_PACKET_COMPONENT(conn_id, freq) src._AddComponent(list( \
+#define MAKE_DEFAULT_RADIO_PACKET_COMPONENT_NETID(conn_id, freq) src._AddComponent(list( \
 		/datum/component/packet_connected/radio, \
 		conn_id, \
 		freq, \
-		("net_id" in src.vars) ? src.vars["net_id"] : null, \
```

Comment:
Could you not just remove the ternary since DM gracefully handles missing associative list indexes? I assume it's the `in` operation that's taking the time here.

Reply:
vars is a special list and accesses are treated as accessing the var itself with :

Reply:
because of course it is

Reply:
Oouuughhh byond. Err okay, could you just move the net ID to be an argument to the macro instead of doing this hacky var list stuff in the first place?

Reply:
maybe, ill see when i get the chance

Reply:
works, just tested, committed, and pushed